pelican/tools/pelican_import.py

```diff
@@ -499,8 +499,11 @@ def get_tumblr_posts(api_key, blogname, offset=0):
                     fmtstr = '<p><a href="%s">via</a></p>\n'
                 source = fmtstr % post.get('source_url')
                 caption = post.get('caption')
-                players = '\n'.join(player.get('embed_code')
-                                    for player in post.get('player'))
+                try:
+                    players = '\n'.join(player.get('embed_code')
+                                        for player in post.get('player'))
+                except TypeError:
```

Comment:
This does work. My question is: is it okay to fail silently, or should there at least be something in the output that says "Video missing" or "Video removed" or something?

Reply:
Good point. There should probably be a line here with something like:
```python
logger.warning(f"Video not imported due to broken embedded video links: {some-appropriate-variable}")
```
What do you think, @willthong?

Reply:
I was actually thinking something in the generated output.

If I'm importing 100 posts, I'm not going to know which one broke, and I'm not looking too closely at warnings.

I'd be willing to work on this, but only after #3215 is merged (to get the unit testing setup).

Reply:
That would indeed be clearer. Thanks for offering to work on this. I just merged 3215. 👍

Reply:
Put up a new PR #3218, based on this one.